### PR TITLE
[Minor-Fix][337] Admin menu in header template.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
Issue #337 
Drupal.org issue: https://www.drupal.org/node/2880723

Steps:
- [x] log in as admin
- [x] go to page [sitename]/header.html
- [x] verify you don't see admin menu
- [x] go to page [sitename]/footer.html
- [x] verify you don't see admin menu
- [x] go to front page or any page 
- [x] verify you can see admin menu

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1249 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1249 |
| Installation process | http://install.openy-dev.ffwua.com/build1249/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1249/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1249/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

File: modules/custom/openy_hf/openy_hf.module [hunk 1]
```diff
@@ -51,6 +51,9 @@ function openy_hf_page_attachments(array &$attachments) {
 function openy_hf_preprocess_html(&$variables) {
   $route = \Drupal::service('current_route_match')->getRouteName();
   if ($route == 'openy_hf.header' || $route == 'openy_hf.footer') {
+    // Remove Admin menu in header and footer template.
+    unset($variables['page_top']['toolbar']);
```
Comment (alexschedrov) [inline modules/custom/openy_hf/openy_hf.module]: Let's wrap into `if` condition. We can't garantee that this element will be always available.
```php
if (isset($variables['page_top']['toolbar'])) {
  unset($variables['page_top']['toolbar']);
}
```

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1255 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1255 |
| Installation process | http://install.openy-dev.ffwua.com/build1255/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1255/

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1255/profiles/contrib/openy/build/reports/behat

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1258 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1258 |
| Installation process | http://install.openy-dev.ffwua.com/build1258/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1258/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1258/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**